modules/gapi/samples/pipeline_modeling_tool/pipeline_builder.hpp

```diff
@@ -362,6 +363,9 @@ void PipelineBuilder::addInfer(const CallParams&  call_params,
     }
 
     pp->pluginConfig(infer_params.config);
+    if (infer_params.out_precision) {
+        pp->cfgOutputPrecision(infer_params.out_precision.value());
+    }
```

Comment:
No multi-output cases supported? What if the user writes such a config? Would it be rejected gracefully?

Reply:
Answered above